meson_options.txt

```diff
@@ -6,4 +6,8 @@ option('man', type : 'combo',
     choices : ['true', 'false'], value : 'true',
     description : 'Build and install man pages')
 
+option('set_coverage', type : 'combo',
```

Comment:
`set_coverage` sounds more like a command. How about `build_with_coverage`?

Reply:
Maybe `with_coverage` would be enough